paf-mvp-demo-express/nodemon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "watch": ["./"],
+  "watch": ["../"],
```

Comment:
This should be a dedicated commit, my bad I was too lazy.
The idea is that since we are using `ts-node` to run the project locally, we need to restart it anytime something changes _in any of the sub-projects_, not just the demo project.